.github/workflows/main.yml

```diff
@@ -0,0 +1,28 @@
+name: Continuous integration
+
+on: [pull_request]
+
+jobs:
+  cypress-run:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** :grey_question: 
 I don't know if you tried this to solve the CI fail-check
 ```yaml
 name: Verify package-lock.json
        run: npm install --package-lock-only
        working-directory: ./ or ./path/to/your/project
```
The package.json and package-lock.json must be sync
<sub><sub> I'm using [conventional comments](https://conventionalcomments.org/url) </sub></sub> ‏

Reply:
In fact, I did that before reading your comment haha. Thanksss btw!